packages/codegen-ui-react/lib/__tests__/__snapshots__/studio-ui-codegen-react-forms.test.ts.snap

```diff
@@ -1675,12 +1681,13 @@ export default function CreateCompositeDogForm(props) {
               modelFields[key] = undefined;
             }
           });
+          const modelFieldsToSave = {
```

Comment:
Can we flip this to create a new object from the fields we want rather than deleting the fields we don't?

Reply:
Done